pkg/controller/combo_controller.go

```diff
@@ -153,6 +157,53 @@ func (c *combinationController) Reconcile(ctx context.Context, req ctrl.Request)
 		Message:            "evaluations successfully processed",
 	})
 
+	// If manifest application is indicated, create generated manifests on-cluster.
+	if combination.Spec.Apply {
+		// update status to indicate processing
+		combination.SetStatusCondition(metav1.Condition{
+			Type:               v1alpha1.TypeInProgress,
+			Status:             metav1.ConditionTrue,
+			Reason:             v1alpha1.ReasonProcessing,
+			LastTransitionTime: metav1.NewTime(time.Now()),
+			Message:            "application of evaluations to the cluster in progress",
+		})
+
+		for _, manifest := range generatedManifests {
+			dec := yaml.NewYAMLOrJSONDecoder(strings.NewReader(manifest), 10)
+			obj := &unstructured.Unstructured{}
+			err := dec.Decode(obj)
+			if err != nil {
+				combination.SetStatusCondition(metav1.Condition{
+					Type:               v1alpha1.TypeInvalid,
+					Status:             metav1.ConditionFalse,
```

Comment:
This should be set to `True` (as in "this _is_ invalid"), right?
```suggestion
					Status:             metav1.ConditionTrue,
```

Reply:
Also, if a previous reconcile set this to `True`, but the user resolved the problem, when do we set it to false?

Reply:
So this probably outlines a misunderstanding I had when writing this. Should every type of Condition be updated during reconciliation steps? Right now, this is just moving through the conditions and overwriting the current status on the list. Should it instead be having all of a set of relevant conditions precent (Invalid, Processing, Processed, etc.) and update the "Status" portion of that through the process?

Reply:
Lots of good detail here: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#typical-status-properties

To me, this is one of the harder things to get right. The general pattern I try to follow is:
1. Ensure every single condition supported by the operator is set before you return from reconcile.
2. Every single condition should be set based on the latest known state for that condition.
3. If you can't derive a definitive True/False the status of a particular condition, it should be set to "Unknown" with a description message about why it could not be determined (usually from an error that occurred somewhere)